content/api/commands/session.md

```diff
@@ -0,0 +1,729 @@
+---
+title: session
+---
+
+Cache and restore [cookies](/api/cypress-api/cookies),
+[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+and
+[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+in order to reduce test setup times.
+
+<Alert type="warning">
+
+The `cy.session()` API is currently experimental, and can be enabled by setting
+the `experimentalSessionSupport` flag to `true` in the Cypress config or by
```

Comment:
Two points here:
1. We also need to add the `experimentalSessionSupport` to the experiments table in https://deploy-preview-4004--cypress-docs.netlify.app/guides/references/experiments#Configuration
2. Within that "description" column of the table we should link back to the cy.session doc.

Reply:
My PR has this addition

Reply:
Done in 4d36c81ef92abd80e606fe20daf9cb2083c47636